index.html

```diff
@@ -8,12 +8,121 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <title>Introducing Karma</title>
 </head>
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+      <div class="LogoNavbox">
+              <a href="#">
+                    
+                  <img
+                      class="karma-logo"
+                      src="img/karma-logo.svg"
+                      alt="The Logo for Karma"
+                  />
+              </a>
 
+              <!-- see https://www.w3schools.com/css/css_navbar.asp -->
+              
+              <nav>
+                <ul class="navigation-list">
+                  <li class="navigation-item">
+                    <a class="navigation-link" id="meetKarma"; href="#">Meet Karma</a>
+                  </li>
+                  <li class="navigation-item">
+                    <a class="navigation-link" href="#">How it Works</a>
+                  </li>
+                  <li class="navigation-item">
+                    <a class="navigation-link" href="#">Store</a>
+                  </li>
+                  <li class="navigation-item">
+                    <a class="navigation-link" href="#">Blog</a>
+                  </li>
+                  <li class="navigation-item">
+                      <a class="navigation-link" href="#">Help</a>
+                    </li>
+                    <li class="navigation-item">
+                      <a class="navigation-link" href="#">Login</a>
+                    </li>
+                    </ul>
+              </nav>
+    </div>
+
+    <div class="mainImage">
+            <div class="centred">
+                <p class="para1">Introducing Karma</p>
```

Comment:
Are there better tags than `p` that represent the role this text is playing in the page? maybe one of the `h` tags would be a better fit?